solidity/random-beacon/test/Groups.test.ts

```diff
@@ -62,91 +62,92 @@ describe("Groups", () => {
 
           expect(storedGroup.groupPubKey).to.be.equal(groupPublicKey)
           expect(storedGroup.activationBlockNumber).to.be.equal(0)
-          expect(storedGroup.members).to.be.deep.equal(members)
+          expect(storedGroup.membersHash).to.be.equal(
+            keccak256(defaultAbiCoder.encode(["uint32[]"], [members]))
```

Comment:
Could this hash calculation be pulled to a util function? E.g. in `test/utils/groups.ts`.

Reply:
5f5de44